mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -19,9 +19,13 @@
 #include <mavros/setpoint_mixin.h>
 #include <eigen_conversions/eigen_msg.h>
 
+#include <message_filters/subscriber.h>
+#include <message_filters/synchronizer.h>
+#include <message_filters/sync_policies/approximate_time.h>
+
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/TwistStamped.h>
-#include <std_msgs/Float64.h>
+#include <mavros_msgs/Float32Stamped.h>
```

Comment:
Since I'm changing this to a custom msg, maybe I should change to `Thrust.msg` only